### PR TITLE
Resetting pr

### PR DESCRIPTION
**Pull request by vishalbabu**
Resetting previous changes to CONTRIBUTING.md so a proper merge can occur.

### CODE REVIEW

**Comment by arumoy-shome** [comment]
Nice! Can you address the changes I had requested in #35?

**Comment by arumoy-shome** [comment]
@vishalbabu I think you forgot to include the new FAQ doc that contains your note in the PR? Can you push it up so we can 🚀 this PR?

**Comment by arumoy-shome** [comment]
Looking good, however there is nothing that links to the new doc that you just created. We should try to make it easier for others to find it, I suggest we add a link to the FAQ doc on the main README file. It should probably be somewhere in the setup section.

Since the doc is markdown you can style it, I have left some comments. Come to think of it, I don't think I have any guidelines on how to write docs so perhaps we should add that? @vishalbabu would you mind opening up an issue for it? We can discuss and develop the guide in the issue once you open it. 😄

File: docs/FAQs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Frequently Asked Questions
```
Comment (arumoy-shome) [inline docs/FAQs.md:1]: This should be a level 1 header.

File: docs/FAQs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+1. During set-up my makefile is not being created. Citing nokogiri error.
+	
+	The error occurs with accessing the Makefile for the nokogiri gem. This occurs when initially installing the UWNerdist bundle. To explain it more clearly here is an explination from Stack overflow;
```
Comment (arumoy-shome) [inline docs/FAQs.md:5]: No need to indent anything as it's markdown.

File: docs/FAQs.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+	For some reason Apple’s Yosemite version of OSX does not have a system accessible installation of libxml2. Nokogiri requires this in order to compile and luckily Xcode has a version of libxml2 bundled with it — we just need to specify it when installing the gem. It’s important to get Nokogiri installed correctly because as of right now Rails 4.2.1.rc4 automatically attempts to install it and you will feel pain.
+
+	So the fix just gives a work around to install nokogiri using the Xcode. Type the following into terminal.
+
+	gem uninstall #
```
Comment (arumoy-shome) [inline docs/FAQs.md:11]: these steps should be in a code block.

Comment (arumoy-shome) [inline docs/FAQs.md:11]: don't use `#` as a placeholder, it will be interpreted as a header by the markdown engine.

**Review by arumoy-shome — CHANGES_REQUESTED** [review]
A few styling changes are required before we can merge this. Looks good otherwise!

**Comment by arumoy-shome** [comment]
@vishalbabu this PR has been stale for a while, any updates on this?

**Comment by vishalbabu** [comment]
@arumoy-shome I just did an update on the repo and my FAQ.md doc is not on under the docs folder anymore. Think this issue can be closed for now.

**Comment by arumoy-shome** [comment]
Huh that's weird, are you on your local branch? You won't see the file if you are on `master` since we didn't merge your PR yet.

**Pull request closed without merging**